build.sbt

```diff
@@ -318,10 +318,18 @@ lazy val commonSettings = ReleasePlugin.extraReleaseCommands ++ Seq(
         <url>http://github.com/fwbrasil/</url>
       </developer>
     </developers>)
-) ++ update2_12
+) ++ `scala 2.12 hacks`
 
-lazy val update2_12 = Seq(
-  scalacOptions -= (
-    if (scalaVersion.value.startsWith("2.12.")) "-Xfatal-warnings" else ""
-  )
-)
+lazy val `scala 2.12 hacks` = Seq(
```

Comment:
nice, it's more granular! I'm still curious how we're going to get rid of the hacks, though. Is it something we can fix?